setup.py

```diff
@@ -182,7 +182,7 @@ def get_extras_require() -> Dict[str, List[str]]:
             "torchvision==0.8.2 ; sys_platform=='darwin'",
             "torchvision==0.8.2+cpu ; sys_platform!='darwin'",
             "torchaudio==0.7.2",
-            "allennlp==1.2.0 ; python_version<'3.8'",
+            "allennlp ; python_version<'3.8'",
```

Comment:
[Question] This is out of the scope of this PR, do we have still problems to execute the AllenNLP related tests in Python 3.8?

Reply:
Thank you for the catch!
d5d5184 removes version constraint in integration section.

Reply:
Thank! LGTM, again.